src/test/java/io/aiven/kafka/connect/http/sender/DefaultHttpSenderTest.java

```diff
@@ -88,7 +89,7 @@ void shouldBuildDefaultHttpRequest() throws Exception {
                     .isPresent()
                     .get(as(InstanceOfAssertFactories.DURATION))
                     .hasSeconds(config.httpTimeout());
-                assertThat(httpRequest.method()).isEqualTo("POST");
+                assertThat(httpRequest.method()).isEqualTo(HttpMethodsType.POST.name());
```

Comment:
We can stick to the string values here, so if enum changes, then tests will fail. Same for new put tests